cves/CVE-2014-0483.yml

```diff
@@ -141,194 +180,181 @@ unit_tested:
     Were automated unit tests involved in this vulnerability?
     Was the original code unit tested, or not unit tested? Did the fix involve
     improving the automated tests?
-
     For code: and fix: - your answer should be boolean.
-
     For the code_answer below, look not only at the fix but the surrounding
     code near the fix in related directories and determine if and was there were unit tests involved for this subsystem. The code
-
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: true
+  fix: true
+  fix_answer: true
 discovered:
   question: |
     How was this vulnerability discovered?
-
     Go to the bug report and read the conversation to find out how this was
     originally found. Answer in longform below in "answer", fill in the date in
     YYYY-MM-DD, and then determine if the vulnerability was found by a Google
     employee (you can tell from their email address). If it's clear that the
     vulenrability was discovered by a contest, fill in the name there.
-
     The automated, contest, and developer flags can be true, false, or nil.
-
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+   2014-08-14, I looked all over the git and the django website but no reaction nor any indication of how
+   or when this vulnerability was found.
+  automated: false
+  contest: false
+  developer: true
 autodiscoverable:
   instructions: |
     Is it plausible that a fully automated tool could have discovered
     this? These are tools that require little knowledge of the domain,
      e.g. automatic static analysis, compiler warnings, fuzzers.
-
     Examples for true answers: SQL injection, XSS, buffer overflow
-
     Examples for false: RFC violations, permissions issues, anything
     that requires the tool to be "aware" of the project's
     domain-specific requirements.
-
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: This was discovered by a developer and no indication of any tools
```

Comment:
This should answer whether or not it is possible that this could be found using a tool, not that it was specifically.